09 News Fixes/configs/ui/maingame_pda_msg_16.xml

```diff
@@ -0,0 +1,17 @@
+<w>
+	<icon_static x="0" y="0" width="41" height="29" stretch="1"/>
+
+	<time_static x="46" y="0" width="42" height="14">
+		<text font="letterica16" color="ui_1"/>
+	</time_static>
+	
+	<caption_static x="75" y="0" width="408" height="14">
+		<text font="letterica16" color="ui_5"/>
+	</caption_static>
+
+	<msg_static x="46" y="20" width="454" height="14" complex_mode="1">
+		<text font="letterica16" color="ui_1"/>
+	</msg_static>
+
+</w>
+
```

Comment:
Same as above file. 32:9 does not read this file, and the Lua doesn't edit it either. Might need this to be changed for other aspect ratios